framework/Distributions.py

```diff
@@ -2499,6 +2499,9 @@ def _handleInput(self, paramInput):
     if self.upperBound is None:
       self.raiseAnError(IOError,' upperBound parameter is needed for LogUniform distribution')
 
+    if self.upperBound < self.lowerBound:
+      self.raiseAnError(IOError,' LogUniform distribution: the upperBound parameter is lower than the lowerBound parameter')
```

Comment:
Do we want to consider swapping them for the user (like your test case had), or do we always want to error out?  I think I generally agree with erring out, but I want to check anyway.